src/common/Promise.ts

```diff
@@ -116,8 +116,8 @@ export class Sink<T> {
     private privPromiseResult: PromiseResult<T> = null;
     private privPromiseResultEvents: PromiseResultEventSource<T> = null;
 
-    private privSuccessHandlers: Array<((result: T) => void)> = [];
-    private privErrorHandlers: Array<(e: string) => void> = [];
+    private privSuccessHandlers: ((result: T) => void)[] = [];
```

Comment:
Lint rules are failing with: ERROR: D:/a/1/s/src/common/Promise.ts:119:34 - Array type using 'T[]' is forbidden for non-simple types. Use 'Array<T>' instead.

Reply:
Thanks, fixed.